From f34523e4732e4f9e98b72717aa4d6dee60e58c3d Mon Sep 17 00:00:00 2001
From: Josef Raschen <josef@raschen.org>
Date: Wed, 10 Jun 2015 21:41:56 +0200
Subject: [PATCH] Environment for Raspberry Pi 1.

---
 include/configs/rpi-common.h | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/include/configs/rpi-common.h b/include/configs/rpi-common.h
index 3121ac9..35a3ea9 100644
--- a/include/configs/rpi-common.h
+++ b/include/configs/rpi-common.h
@@ -179,8 +179,16 @@
 #define CONFIG_EXTRA_ENV_SETTINGS \
 	ENV_DEVICE_SETTINGS \
 	ENV_MEM_LAYOUT_SETTINGS \
-	BOOTENV
+	BOOTENV \
+    "fdtfile=zImage-bcm2708-rpi-b-plus.dtb\0" 
 
 #define CONFIG_BOOTDELAY 2
 
+#define CONFIG_BOOTFILE "zImage"
+#define CONFIG_BOOTARGS "earlyprintk console=tty0 console=ttyAMA0 root=/dev/mmcblk0p2 rootfstype=ext4 rootwait noinitrd"
+#define CONFIG_BOOTCOMMAND \
+    "fatload mmc 0:1 ${kernel_addr_r} ${bootfile}; "\
+    "fatload mmc 0:1 ${fdt_addr_r} ${fdtfile}; "\
+    "bootz ${kernel_addr_r} - ${fdt_addr_r}"
+
 #endif
-- 
2.1.4

