Index: git/qtwayland.pro
===================================================================
--- git.orig/qtwayland.pro	2015-06-28 18:05:09.458018537 -0300
+++ git/qtwayland.pro	2015-06-28 18:44:27.781919359 -0300
@@ -1,13 +1,11 @@
 load(configure)
 qtCompileTest(wayland)
-qtCompileTest(xkbcommon)
 qtCompileTest(wayland_cursor)
 qtCompileTest(wayland_scanner)
 qtCompileTest(wayland_egl)
 qtCompileTest(egl)
 qtCompileTest(brcm_egl)
 qtCompileTest(glx)
-qtCompileTest(xcomposite)
 qtCompileTest(drm_egl_server)
 qtCompileTest(libhybris_egl_server)
 
